mbf_abstract_nav/src/abstract_planner_execution.cpp

```diff
@@ -106,10 +106,14 @@ typename AbstractPlannerExecution::PlanningState AbstractPlannerExecution::getSt
   return state_;
 }
 
-void AbstractPlannerExecution::setState(PlanningState state)
+void AbstractPlannerExecution::setState(PlanningState state, bool signalling)
 {
   boost::lock_guard<boost::mutex> guard(state_mtx_);
   state_ = state;
+
+  // some states are quiet, most aren't
+  if(signalling)
```

Comment:
missing space after if

Reply:
would it make sense to add a clang-format file so we use common formatting?

Reply:
yes,,,  I will